cmd/bosun/sched/check.go

```diff
@@ -256,8 +266,26 @@ func (s *Schedule) CheckExpr(T miniprofiler.Timer, rh *RunHistory, a *conf.Alert
 		rh.Events[ak] = &Event{
 			Status: StError,
 		}
+		return nil, err
+	}
+	return results, err
+}
+
+func (s *Schedule) CheckExpr(T miniprofiler.Timer, rh *RunHistory, a *conf.Alert, e *expr.Expr, checkStatus Status, ignore expr.AlertKeys) (alerts expr.AlertKeys, err error) {
```

Comment:
It looks like checking to see if results are in ignore got removed during an intervening diff.